opencti-platform/opencti-front/src/private/components/observations/indicators/IndicatorLine.js

```diff
@@ -45,123 +43,109 @@ const styles = (theme) => ({
-                style={{ width: dataColumns.valid_until.width }}
-              >
-                {fd(node.valid_until)}
-              </div>
-              <div
-                className={classes.bodyItem}
-                style={{ width: dataColumns.objectMarking.width }}
-              >
-                <ItemMarkings
-                  markingDefinitions={pathOr(
-                    [],
-                    ['objectMarking', 'edges'],
-                    node,
-                  )}
-                  limit={1}
-                  variant="inList"
-                />
-              </div>
-            </div>
+        <Checkbox
+          edge="start"
+          checked={
+            (selectAll && !(node.id in (deSelectedElements || {})))
+            || node.id in (selectedElements || {})
           }
+          disableRipple={true}
         />
-        <ListItemIcon classes={{ root: classes.goIcon }}>
-          <KeyboardArrowRight />
-        </ListItemIcon>
-      </ListItem>
-    );
-  }
-}
-
-IndicatorLineComponent.propTypes = {
-  dataColumns: PropTypes.object,
-  node: PropTypes.object,
-  classes: PropTypes.object,
-  fd: PropTypes.func,
-  nsdt: PropTypes.func,
-  onLabelClick: PropTypes.func,
-  onToggleEntity: PropTypes.func,
-  selectedElements: PropTypes.object,
-  deSelectedElements: PropTypes.object,
-  selectAll: PropTypes.bool,
+      </ListItemIcon>
+      <ListItemIcon classes={{ root: classes.itemIcon }}>
+        <ShieldSearch />
+      </ListItemIcon>
+      <ListItemText
+        primary={
+          <div>
+            <div
+              className={classes.bodyItem}
+              style={{ width: dataColumns.pattern_type.width }}
+            >
+              <ItemPatternType variant="inList" label={node.pattern_type} />
+            </div>
+            <div
+              className={classes.bodyItem}
+              style={{ width: dataColumns.name.width }}
+            >
+              {node.name}
+            </div>
+            <div
+              className={classes.bodyItem}
+              style={{ width: dataColumns.objectLabel.width }}
+            >
+              <StixCoreObjectLabels
+                variant="inList"
+                labels={node.objectLabel}
+                onClick={onLabelClick.bind(this)}
+              />
+            </div>
+            <div
+              className={classes.bodyItem}
+              style={{ width: dataColumns.created.width }}
+            >
+              {nsdt(node.created)}
+            </div>
+            <div
+              className={classes.bodyItem}
+              style={{ width: dataColumns.creator.width }}
+            >
+              {node.creator.name}
+            </div>
+            <div
+              className={classes.bodyItem}
+              style={{ width: dataColumns.valid_until.width }}
+            >
+              {fd(node.valid_until)}
+            </div>
+            <div
+              className={classes.bodyItem}
```

Comment:
I have a problem with the display of this property
![image](https://user-images.githubusercontent.com/53513584/207544770-3d1b5e3b-e2d5-4699-a776-a87b0b43c345.png)

Reply:
Done !